state/event_table.go

```diff
@@ -128,12 +128,10 @@ func NewEventTable(db *sqlx.DB) *EventTable {
 		membership TEXT,
 		is_state BOOLEAN NOT NULL, -- is this event part of the v2 state response?
 		event BYTEA NOT NULL,
+		-- True iff the previous timeline event is not known to the proxy.
```

Comment:
Definitely add some prose about the fact that we check `limited`, otherwise in the common case of 1 single new event in a sync v2 response, do we know the previous event?